ui/imports/shared/stores/send/TransactionStore.qml

```diff
@@ -157,14 +157,24 @@ QtObject {
     }
 
     function collectibleToSelectorCollectible(collectible) {
+        var groupId = collectible.collectionUid
+        var groupName = collectible.collectionName
+        var itemType = Constants.CollectiblesNestedItemType.Collectible
+        if (collectible.communityId !== "") {
+            groupId = collectible.communityId
+            groupName = collectible.communityName
+            itemType = Constants.CollectiblesNestedItemType.CommunityCollectible
+        }
         return {
             uid: collectible.uid,
             chainId: collectible.chainId,
             name: collectible.name,
             iconUrl: collectible.imageUrl,
-            collectionUid: collectible.collectionUid,
-            collectionName: collectible.collectionName,
-            isCollection: false
+            groupId: groupId,
+            groupName: groupName,
+            tokenType: collectible.tokenType,
+            itemType: itemType,
+            count: 1 // TODO: Properly handle count
```

Comment:
can we create a follow up task for this TODO please? or link to an existing github issue?

Reply:
Here https://github.com/status-im/status-desktop/issues/14251

Reply:
I meant if the link to that issue can be added to the code :)